src/ssl/ssl_openssl_impl.cpp

```diff
@@ -560,10 +560,8 @@ CassError OpenSslContext::add_trusted_cert(const char* cert, size_t cert_length)
   int num_certs = 0;
 
   // Iterate over the bio, reading out as many certificates as possible.
-  for (X509* cert = PEM_read_bio_X509(bio, NULL, pem_password_callback, NULL);
-       cert != NULL;
-       cert = PEM_read_bio_X509(bio, NULL, pem_password_callback, NULL))
-  {
+  for (X509* cert = PEM_read_bio_X509(bio, NULL, pem_password_callback, NULL); cert != NULL;
+       cert = PEM_read_bio_X509(bio, NULL, pem_password_callback, NULL)) {
```

Comment:
Formatting change which was leading to Jenkins build failures on [PR 522](https://github.com/datastax/cpp-driver/pull/522).  The format check was only failing on Ubuntu Bionic instances so it's worth validating that we're okay on the other platforms as well... last thing we need is to introduce a failure somewhere else via this change.